.github/workflows/unit_test.yml

```diff
@@ -0,0 +1,24 @@
+name: Unit_test
+
+on:
```

Comment:
Can be replace on `on: [push]` as here:

https://github.com/onlyoffice-testing-robot/onlyoffice_logger_helper/blob/master/.github/workflows/ruby.yml#L2

Same meaning for our projects but shorted